packages/app/src/pages/over.tsx

```diff
@@ -1,23 +1,34 @@
-import Head from 'next/head';
-import { MaxWidth } from '~/components-styled/max-width';
 import { FCWithLayout, getLayoutWithMetadata } from '~/domain/layout/layout';
+import { MaxWidth } from '~/components-styled/max-width';
 import { PortableText } from '~/lib/sanity';
+import Head from 'next/head';
 import siteText from '~/locale/index';
+import { createGetStaticProps } from '~/static-props/create-get-static-props';
 import {
   createGetContent,
   getLastGeneratedDate,
 } from '~/static-props/get-data';
-import { createGetStaticProps } from '~/static-props/create-get-static-props';
-import styles from './over.module.scss';
+import { Image } from '~/components-styled/image';
 import { PortableTextEntry } from '@sanity/block-content-to-react';
+import { SanityImageProps } from '~/types/cms.d.ts';
```

Comment:
nit; The ts part will be resolved just like other ts files.

```suggestion
import { SanityImageProps } from '~/types/cms.d';
```

Reply:
You can leave me `.d` part as well